### PR TITLE
[CAES] Changed decryption process from lists to items

### PR DESCRIPTION
**Pull request by dspiridonov4xxi**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -35,20 +35,16 @@ const publicRuntimeConfig = {
   MAX_UPLOADING_FILE_SIZE: process.env.MAX_UPLOADING_FILE_SIZE || '256KB',
   TOTAL_MAX_UPLOADING_FILES_SIZES: process.env.TOTAL_MAX_UPLOADING_FILES_SIZES || '5M',
   LENGTH_KEY: process.env.LENGTH_KEY || 2048,
+  WORKER_DECRYPTION_BUFFER_SIZE: process.env.WORKER_DECRYPTION_BUFFER_SIZE || 10,
```
Comment (beshkenadze) [inline next.config.js]: Why we have constant variable WORKER_DECRYPTION_BUFFER_SIZE and environment variable WORKER_DECRYPTION_BUFFER_SIZE at one time?

File: common/constants.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ export const {
 export const PORTAL_ID = 'portal';
 export const DEFAULT_IDLE_TIMEOUT = 5 * 60 * 1000;
 export const MAX_SIZE_RANDOM_BUFFER = 60000;
+export const WORKER_DECRYPTION_BUFFER_SIZE = 10;
```
Comment (beshkenadze) [inline common/constants.js]: Why we don't use navigator.hardwareConcurrency API?

Comment (dspiridonov4xxi) [reply common/constants.js]: It's not a count of cpu cores and pool size. It's a buffer for decrypted items inside worker.

File: next.config.js (hunk 1 above)
Comment (dspiridonov4xxi) [reply next.config.js]: My mistake. I'll fix it

**Pull request merged**